### PR TITLE
change in readme

### PR DESCRIPTION
**Pull request by relferreira**
changes in hello from readme

### CODE REVIEW

**Comment by relferreira** [comment]
teste comment in pull request

**Comment by relferreira2** [comment]
teste comment

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Test
 
-hello
```
Comment (relferreira2) [inline README.md]: sure?

Comment (relferreira) [inline README.md]: yes

Comment (relferreira2) [inline README.md]: ok

Comment (relferreira) [inline README.md]: teste

Comment (relferreira) [inline README.md]: teste2

Comment (relferreira) [inline README.md]: teste3

Comment (relferreira) [inline README.md]: teste4

**Pull request merged**